src/planscape/forsys/spatial_clustering.R

```diff
@@ -0,0 +1,80 @@
+suppressMessages({
+    library(sf)
+    # TODO: switch?
+    library(tidyverse)
+})
+
+normalize_values <- function(x) {
+  (x - min(x)) / (max(x) - min(x))
+}
+
+cluster_cells_to_stands <- function(
+    stand_data,
+    stand_id_field = "stand_id",
+    stand_area_field = "area",
+    stand_cost_field = "cost",
+    scenario_priorities = c(
+      "priority1",
+      "priority2",
+      "priority3",
+      "priority4"),
+    geo_weight = 2,
+    avg_cells_per_cluster = 10,
+    seed = 42
+) {
+
+  # Creating data for clustering
+  stand_data <- stand_data %>%
+    # select(
+    #   {{stand_id_field}},
+    #   {{stand_area_field}},
+    #   {{stand_cost_field}},
+    #   one_of({{scenario_priorities}})) %>%
```

Comment:
Please remove.

Reply:
Done.

The question remains, what do we want to do with cells that contain NA values for some metrics? I don't think we currently have implemented a business logic that makes a conscious decision about this, do we?